instructor/ui/instructor.js

```diff
@@ -163,6 +163,30 @@ $(function(){
     $("#id_has_nuclear_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
     $("#id_has_cytoplasmic_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
 
+    /**
+     * Western Blotting: band intensity
+     */
+    var sliders = $(".intensity_slider");
+    for(var i=0; i<sliders.length;i++){
+        var $slider=$(sliders[i]);
+        var intensity_dict={0:'', 0.5:'low', 1:'med', 1.5:'high', 2:'extra high'};
```

Comment:
Actually it looks like there is a floating point comparison problem. If you set the sliders to different levels, save it, go to a different page and come back, some of the text shows up and some doesn't. When the floating point keys are put into an object they're converted to strings, so the `2` is actually stored as `"2"`. In the `value` attribute it's being stored as a `"2.0"` so the two strings don't compare as being equal.

It's probably best to just use a series of `if` statements, something like

    if (intensity >=0 && intensity < 0.75) ...
    else if (intensity >= 0.75 && intensity < 1.25) ...

to avoid the problems with floating point comparison